src/App.js

```diff
@@ -36,19 +43,37 @@ const donations = [
   },
 ];
 
-function App() {
-  return (
+
+
+
+// function App () {
+  const totalDonated = donations.reduce((acc, ele)=> (acc += ele.amount), 0);
+
+  export default class App extends React.Component {
```

Comment:
Please avoid using class components. Since we will be using hooks moving forward we want to stick to functional components